tests-and-examples/Cargo.toml

```diff
@@ -6,8 +6,9 @@ autoexamples = false
 publish = false
 
 [features]
-default = ["protobuf-codec"]
+default = ["protobufv3-codec"]
```

Comment:
I suggest to make "protobuf-codec" default instead just like other crates in the repo.

Reply:
Good catch, thanks